Backend/src/main/java/com/team33/backend/comment/domain/Comment.java

```diff
@@ -50,19 +53,82 @@ public class Comment extends CommonEntity {
     @JoinColumn(name = "issue_id", nullable = false)
     private Issue issue;
 
-    @OneToMany(mappedBy = "comment", cascade = CascadeType.REMOVE)
-    private List<Emoji> emojis = new ArrayList<>();
+    @OneToMany(mappedBy = "comment", cascade = {CascadeType.PERSIST, CascadeType.REMOVE})
+    private Set<Emoji> emojis = new HashSet<>();
 
-    public Comment(String content, List<Image> images, Member member, Issue issue) {
+    public Comment(String content) {
         this.content = content;
-        this.images = images;
+        this.deleted = initDeleted();
+    }
+
+    public Comment(String content, Member member, Issue issue) {
+        validateContent(content);
+        this.content = content;
+        validateMember(member);
         this.member = member;
+        validateIssue(issue);
         this.issue = issue;
+        this.deleted = initDeleted();
     }
 
     protected Comment() {
     }
 
+    private Deleted initDeleted() {
+        return new Deleted();
+    }
+
+    public void registMember(Member member){
+        this.member = member;
+    }
+
+    public void registIssue(Issue issue){
+        this.issue = issue;
+    }
+
+    private void validateContent(String content) {
+        if (content == null || content.isBlank()) {
+            throw new IllegalArgumentException("내용을 입력해주세요.");
+        }
+    }
+
+    private void validateMember(Member member) {
+        if (member == null) {
+            throw new IllegalArgumentException("회원정보가 존재하지 않습니다.");
+        }
+    }
+
+    private void validateIssue(Issue issue) {
+        if (issue == null) {
+            throw new IllegalArgumentException("이슈정보가 존재하지 않습니다.");
+        }
+    }
+
+    public void editComment(String content) {
+        validateContent(content);
+        this.content = content;
+    }
+
+    public void addImage(Image image) {
+        if (image != null) {
+            this.images.add(image);
+            image.addImageToComment(this);
+        }
+    }
+
+    public void addImages(List<Image> images) {
+        for (Image image : images) {
+            if (image != null) {
+                this.images.add(image);
+                image.addImageToComment(this);
+            }
+        }
```

Comment:
list로 받는 객체는 null 체크 하지 않아도 괜찮을 것 같습니다~
굳이 사용한다면 아래처럼 바로 위에 만들어놓은 메서드 사용해도 좋겠네요.

```suggestion
        for (Image image : images) {
            addImage(image)
        }
```